_topic_maps/_topic_map.yml

```diff
@@ -3207,14 +3207,14 @@ Topics:
   - Name: Virtual machine networking
     Dir: vm_networking
     Topics:
-    - Name: Using the default pod network with OpenShift Virtualization
-      File: virt-using-the-default-pod-network-with-virt
+    - Name: Configuring the default pod network with OpenShift Virtualization
```

Comment:
Please revert the names of these files. It is much better to change only the title. If you change the file name, we need to set up redirects to avoid broken links.

Reply:
Done.

Reply:
Am I missing something? I still see a change in this line.